securedrop_client/config.py

```diff
@@ -0,0 +1,19 @@
+import json
+import os
+
+
+class Config:
+
+    CONFIG_NAME = 'config.json'
+
+    def __init__(self, journalist_key_fingerprint: str) -> None:
+        self.journalist_key_fingerprint = journalist_key_fingerprint
+
+    @classmethod
+    def from_home_dir(cls, sdc_home: str):
+        full_path = os.path.join(sdc_home, cls.CONFIG_NAME)
+        with open(full_path) as f:
+            json_config = json.loads(f.read())
+        return Config(
```

Comment:
> telling the user that journalist_key_fingerprint

Hmm. So it might make sense to just do this at the call time and not at app start. We don't need the GPG key for offline mode, so maybe just "hey your app is not configured, please contact your admin."

Since the above requires a fair bit of design and work, can we move that to a separate ticket to keep this tightly scoped?

Reply:
sure - want to add a followup issue for this?

Reply:
Yeah, I'll add that when this gets merged.